src/types/events.ts

```diff
@@ -0,0 +1,4 @@
+export enum EventsVariant {
```

Comment:
Good point, I say we keep it as it is for now just to keep the consistency, however it would be good to rename all of these to avoid confusion in the future.

Reply:
Rename now what you can. Put into constant what you can't.
Example: name your types and variables experience, but use `COLLECTION_EXPERIENCE="events"` to pass to DB queries.

Also, please do avoid enums, they're hard to deal with since it's TS trying to add C# thing by compiling into a complex JS. The same can be accomplished by:

`export type ExperiencePlace = "world" | "space"; `

Reply:
Oh, you can also make it a branded string. More info at  https://github.com/sparkletown/sparkle/blob/staging/src/types/id.ts